packages/babel-types/src/definitions/experimental.js

```diff
@@ -10,6 +10,8 @@ import {
   classMethodOrDeclareMethodCommon,
 } from "./es2015";
 
+defineType("ArgumentPlaceholder", {});
```

Comment:
We should also update the `CallExpression` definition to accept placeholder arguments

Reply:
@nicolo-ribaudo You mean in `/packages/babel-types/src/definitions/core.js` I have to add another assert node type like this:
```
defineType("CallExpression", {
  visitor: ["callee", "arguments", "typeParameters", "typeArguments"],
  builder: ["callee", "arguments"],
  aliases: ["Expression"],
  fields: {
    callee: {
      validate: assertNodeType("Expression"),
    },
    arguments: {
      validate: chain(
        assertValueType("array"),
        assertEach(
          assertNodeType("Expression", "SpreadElement", "JSXNamespacedName", "ArgumentPlaceholder"),
        ),
      ),
    },
...
```